npc/ipset/ipset.go

```diff
@@ -71,6 +74,9 @@ func New(logger *log.Logger) Interface {
 
 func (i *ipset) Create(ipsetName Name, ipsetType Type) error {
 	args := []string{"create", string(ipsetName), string(ipsetType)}
+	if ipsetType == ListSet && i.maxListSize > 0 {
+		args = append(args, fmt.Sprintf("size %d", i.maxListSize))
```

Comment:
`fmt.Sprintf("size %d", i.maxListSize)` should be split into two elements, i.e. "size", and `fmt.Sprintf("%d", i.maxListSize)`.